File: php/features.php
<?php
declare(strict_types=1);

// TarotCardMeanings SDK feature factory

require_once __DIR__ . '/feature/BaseFeature.php';
require_once __DIR__ . '/feature/TestFeature.php';


class TarotCardMeaningsFeatures
{
    public static function make_feature(string $name)
    {
        switch ($name) {
            case "base":
                return new TarotCardMeaningsBaseFeature();
            case "test":
                return new TarotCardMeaningsTestFeature();
            default:
                return new TarotCardMeaningsBaseFeature();
        }
    }
}

File: py/features.py
# TarotCardMeanings SDK feature factory

from feature.base_feature import TarotCardMeaningsBaseFeature
from feature.test_feature import TarotCardMeaningsTestFeature


def _make_feature(name):
    features = {
        "base": lambda: TarotCardMeaningsBaseFeature(),
        "test": lambda: TarotCardMeaningsTestFeature(),
    }
    factory = features.get(name)
    if factory is not None:
        return factory()
    return features["base"]()
